pages/workshops.json

```diff
@@ -7,16 +7,19 @@ permalink: /workshops.json
   "features": [
   {% for workshop in site.data.amy.workshops_current %}
     {% unless workshop.longitude == NULL or workshop.latitude == NULL %}
-    {
-      "type": "Feature",
-      "geometry": {
-        "type": "Point",
-        "coordinates":[{{workshop.longitude}},{{workshop.latitude}}]
-      },
-      "properties": {
-        "marker-color": "#2b3990",
-        "details": "<a href='{{workshop.url}}'>{{workshop.venue}}: {{workshop.humandate}}</a>"
+      {% unless forloop.first %},{% endunless %}
```

Comment:
What happens if the first workshop in workshops_current isn't displayed because it's missing lat/long, but the second one is displayed because it has lat/long? If I read this correctly, we'll have a comma before the first entry in the GeoJSON file, which isn't valid. It's the inverse of the problem my (broken) initial solution had (I think).